state/executor.go

```diff
@@ -1354,6 +1360,17 @@ func (t *Transition) RevertToSnapshot(snapshot int) error {
 	return nil
 }
 
+// PopulateAccessList populates access list based on the provided access list
+func (t *Transition) PopulateAccessList(acl types.TxAccessList) {
```

Comment:
```suggestion
func (t *Transition) PopulateAccessList(from types.Address, to *types.Address, acl types.TxAccessList) {
    t.accessList.PrepareAccessList(from, to, t.precompiles.Addrs, acl)
}
```

Reply:
What about adding to journal? I think we need it, because we are reverting to snapshot from the benchmark tests. Do you agree? Therefore it would make more sense using `AddAddressToAccessList` and `AddSlotToAccessList` (though I agree to add `from` and `to` addresses in this function as well).

Reply:
But this is a setup that runs before the test itself. Basically, what you are doing here is saying prepare the acl with some predefine data, and that is the starting point, so I don't think you need the journal here. The same thing is done in the regular state tests.

Reply:
Not really, I'm invoking PopulateAccessList multiple time on the same transition (https://github.com/Ethernal-Tech/blade/blob/4afb72d4f5c5fc9143047560e9acb6881e881b4b/tests/evm_benchmark_test.go#L136) and at the end of the for loop, reverting to the snapshot (https://github.com/Ethernal-Tech/blade/blob/4afb72d4f5c5fc9143047560e9acb6881e881b4b/tests/evm_benchmark_test.go#L154).

Anyway, I can try without journaling, and let's see how it turns out.

Reply:
Fixed https://github.com/Ethernal-Tech/blade/pull/142/commits/fc7c0a0bde5c6aee10c7c51ce7da01ecc02165e0 (populating access list in advance, which means that I don't need the `ActivePrecompiles` function).